### PR TITLE
Translation train on MultiWOZ (Chinese) nad CrossWOZ (English) of SUMBT

### PR DESCRIPTION
**Pull request by function2-llx**
Accuracy is 26% and 16% respectively. Used as baselines for DSTC-9 for the moment, refine and fix some issues later if possible and necessary. 
I don't think it's a good idea to download pre-trained models automatically in the code (because of the unstable network), so I'd like to require users download them mannualy [here](https://github.com/function2-llx/ConvLab-2/releases).

### CODE REVIEW

**Review by zqwerty — CHANGES_REQUESTED** [review]
1. add standard evaluation code in `dst/evaluate.py` which can be used for many models once using the DST class **update** interface. Should evaluate not only machine-translated test set but also human-translated dev set (see `data/[multiwoz_zh|crosswoz_en]`).
2. upload the pre-trained baseline model to azure
3. add a clear instruction for cross-lingual dst baseline
4. explain where the database and ontology come from

**Review by zqwerty — CHANGES_REQUESTED** [review]
- add azure links of trained SUMBT on README
- change data_key=`human` to `human_val` in evaluation script, dataloader, and README
- add descriptions for where the ontology come from

**Review by zqwerty — APPROVED** [review]
